Android/Kotlin/config.json

```diff
@@ -20,14 +20,14 @@
 			"groupId" : "org.jetbrains.kotlin",
 			"artifactId" : "kotlin-stdlib",
 			"version" : "1.5.20",
-			"nugetVersion" : "1.5.20",
+			"nugetVersion" : "1.5.20.1",
```

Comment:
The last version on Nuget is 1.5.21, this is because the version 1.5.20 and 1.5.21 landed in the wrong order, so the last commit was 1.5.20, but maybe you want the 1.5.21 version, I'm not sure 🙂

Reply:
Ick, I don't see anywhere in our commit history where it was set to `1.5.21` which scares me.

I just removed doing new packages in this PR, and this fix will get published with https://github.com/xamarin/XamarinComponents/pull/1259.

Thanks for the callout!